### DISCUSSION TITLE
Ability to disable "Always on top"

### DISCUSSION THREAD

**Discussion by bwbishop** [SauceLLC/sauce4zwift#47] [Q&A]
Is there a way to disable "Always On Top" for sauce? When I'm streaming, I bring in each sauce windows as a window capture and then race with OBS on my main (only) monitor. I don't need sauce on top because it's captured and already on my screen, but if I minimize the sauce window, it's no longer captured in OBS.

How do I work around this for streaming in OBS? I only have one monitor on my streaming rig, and that is run in full-screen preview mode.

**Comment by mayfield** [comment]
Hi Brian,

I'm not an OBS expert, but the discord server is literally full of them.  I
bet someone there has already figured this out or has some good
suggestions.  The link should be on the patreon page near the top or in the
connected apps section.

Let me know if you don't find an answer there.

Justin
